stratum/public/proto/p4_role_config.proto

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022-present Open Networking Foundation
+// Copyright 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+// TODO(max)
+// The P4RoleConfig contains ...
+
+syntax = "proto3";
+
+option cc_generic_services = false;
+
+package stratum;
+
+import "p4/v1/p4runtime.proto";
+
+
+// The P4RoleConfig message conists of these fields:
+//  accessible_p4_ids - a list of the
+//  packet_in_filters -
+//  receives_packet_ins -
+message P4RoleConfig {
+  repeated uint32 accessible_p4_ids = 1;
```

Comment:
shall we split in readable/writable ?

Reply:
We agreed to split by "mutually exclusive" and "shared" attribute.